runtime/lua/vim/shared.lua

```diff
@@ -553,7 +553,7 @@ end
 ---@param t table List-like table
 ---@return table Flattened copy of the given list-like table
 function vim.tbl_flatten(t)
-  vim.deprecate('vim.tbl_flatten', 'vim.iter(…):flatten():totable()', '0.13')
+  vim._deprecate('vim.tbl_flatten', 'vim.iter(…):flatten():totable()', '0.12')
```

Comment:
I think we wanted to extend its life until 0.13? #28780

Reply:
Oops, thanks.